packages/dom/test/unit/lib/fixtures/haveClassTestComponent.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from "react";
+
+export function HaveClassTestComponent(): ReactElement {
+    return (
+      <div data-testid="classTest">
```

Comment:
Do we really need the `data-testid`? The `getByTestID(..)` query should be our last resort when finding elements. For instance, this element can be found with the query below instead:
```ts
getByText("Test text inside a div")
```